app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: [float, int]) -> any:
```

Comment:
please, don't use `any` yupe annotation. Here returns nothing: `-> None`. Fix it everywhere -- remove `any` type annotation